kong/db/strategies/postgres/init.lua

```diff
@@ -987,6 +987,10 @@ function _M.new(connector, schema, errors)
     insert(upsert_expressions, ttl_escaped .. " = "  .. "EXCLUDED." .. ttl_escaped)
   end
 
+  if kong.configuration.log_level == "debug" then
+    insert(select_expressions, "txid_current() as _pg_transaction_id")
```

Comment:
Do we even need this if we can just call `global.get_current_transaction_id` in https://github.com/Kong/kong/blob/master/kong/init.lua#L1854-L1891

Reply:
The intention here is to avoid the extra database roundtrip and the unnecessary creation of another empty transaction.  Probably not that big of a win, in particular now that the feature is behind the debug log setting, but overall we're not keen on slowing down our tests either.

Reply:
Yes, I understood the reason. But one pg call should not make much difference I think. The extra roundtrip would only happen with admin api, when there was a write to db? And only when ran with debug. So I think it is pretty well covered already. The change here potentially also affects the proxy side?